### PR TITLE
[PR] Exporting default types properly so it matches the javascript file.

### LINKED ISSUE #411: v6.0.1  broken with  `TypeError: AWS.mock is not a function`
Hello Team, 
i updated aws-sdk-mock to v6.0.1 but when I'm trying to use it i have a `TypeError: AWS.mock is not a function` errors  throwed.

my implementation is simple (plain javascript) and straight forward: 

```js 
const AWS = require('aws-sdk-mock')

AWS.mock('DynamoDB.DocumentClient', 'batchWrite', (params, callback) => {
	mockBatchWrite(params)
	callback(null, true)
})
```

Any one could help ?

### PR DESCRIPTION
**Pull request by LuchoTurtle**
fixes #411 

The problem was related to how the `default export` was being treated from Typescript to Javascript. They didn't match, so what happened is that, after transpiling the Typescript to Javascript, the `js` files would have a different way of exporting the default `AWS` variable than what was described in the Typescript files. Read https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseExportDefault.md for a comprehensive overview.

This PR fixes this and changes the way `AWS` is exported to `export = AWS` (instead of `export default AWS`).

In addition to this, the bundle configuration from `tsup` is now explicit.

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
@nelsonic take a look and approve when possible. I've tested the output bundle (the bundled `.js` files) in the project https://github.com/aannoune/aws-sdk-mock-issue411 and it's working now.

After approval, I'll publish and merge.

**Review by nelsonic — DISMISSED** [review]
@LuchoTurtle code/updates look good. :shipit:

**Pull request merged**
